trees/treetopologyex.11.cpp

```diff
@@ -102,5 +102,7 @@ struct tree_topology_ex : tree_topology {
 int main() {
     tree_topology_ex tt({{0, 1}, {1, 2}, {1, 3}}, 3);
 
+    cout << tt.kth(2, 3, 1);
```

Comment:
Printing a value doesn't really do anything towards the result of the test. You must compare it with the expected value and `return 1` if they're different.

Another idea: Since the tree in this test case is quite small, let's add a different one with say, 10 nodes, pick some two vertices whose LCA is not 0, and test every vertex on the path between them by calling `kth`.